.github/workflows/build-test.yml

```diff
@@ -45,17 +45,17 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install nox
+          pip install -e .[tests]
       - name: List installed packages
         run: pip list
-      # Need tags for setuptools_scm to provide a proper version
+        # Need tags for setuptools_scm to provide a proper version
       - name: Fetch git tags
         run: git fetch origin 'refs/tags/*:refs/tags/*'
         # For now i made a separate tests run so we can see each version
         # build via github actions. There may be a better way to do this.
-      - name: Run tests with pytest & nox
+      - name: Run tests with pytest
         run: |
-          nox -s tests-${{ matrix.python-version }}
+          pytest --cov=src/stravalib/ --cov-report=xml:coverage.xml --cov-report=term tests/
```

Comment:
for some reason the coverage report is FINE when i run it outside of nox. let's see if that is the same in CI.